torch/csrc/jit/script/module.h

```diff
@@ -57,34 +57,19 @@ using ModuleLookup =
     std::function<std::shared_ptr<Module>(const std::vector<std::string>&)>;
 
 struct TORCH_API Method {
-  Method(Module* owner, Function* function);
+  Method(Module* owner, const std::shared_ptr<Function>& function);
```

Comment:
so someone else now owns a function aside from `Method`?

Reply:
for now yes, Michael is working on our final state for Function ownership which will make this stuff more clear.